shared/actions/config/index.js

```diff
@@ -67,7 +67,25 @@ export function getExtendedConfig (): AsyncAction {
   }
 }
 
-export function getCurrentStatus (): AsyncAction {
+let bootstrapSetup = false
+export function bootstrap (): AsyncAction {
+  return dispatch => {
+    if (!bootstrapSetup) {
+      bootstrapSetup = true
+      console.log('Registered bootstrap')
+      engine.listenOnConnect('bootstrap', () => {
+        console.log('Bootstrapping')
+        dispatch(bootstrap())
+      })
+    } else {
+      Promise.all([dispatch(getCurrentStatus()), dispatch(getExtendedStatus())]).then(() => {
+        dispatch(navBasedOnLoginState())
+      })
```

Comment:
I just mean add a `.catch` like how startup did to log a bootstrapping error in `getCurrentStatus` or `getExtendedStatus`